### ISSUE TITLE
The named manifest is not known to the registry.

### ISSUE THREAD

**Issue by lucaspinheiro1** [goharbor/harbor-helm#1895]
I can't get around an issue with my harbor installation, where the replication rule from Jfrog Artifactory to Harbor always give the same error for all images I'm trying to pull: "The named manifest is not known to the registry." Does someone know how could I troubleshoot this ?

I'm using 1.16.0 version.

**Comment by MinerYang** [comment]
Could you manually pull this specific image? And which storage you are using?

**Comment by lucaspinheiro1** [comment]
By the way, I found this unreplied thread on stackoverflow, which talk about my issue: https://stackoverflow.com/questions/78793545/replication-from-artifactory-got-problem-with-manifest-unknown

What do you think about this question ?
`My source manifest.json schema is 2. Does it have impact on replication since I can read: [Docs](https://goharbor.io/docs/2.0.0/administration/configuring-replication/create-replication-rules/) that you can not replicate artifacts with manifest list from 2.0 to 1.x. Does it mean that replication supports only manifests schema version 1?`

**Comment by rivolity** [comment]
@lucaspinheiro1 
Have you solved your problem, we encontre the same issue when replicating from jfrog to harbor.

**Comment by lucaspinheiro1** [comment]
@rivolity 
No, we couldn't identify the issue, so we used another software.
